runner/app/routes/llm_generate.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import os
+from typing import Annotated, Optional, List
+from fastapi import APIRouter, Depends, Form, status, Request
+from fastapi.responses import JSONResponse, StreamingResponse
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+from app.dependencies import get_pipeline
+from app.pipelines.base import Pipeline
+from app.routes.util import HTTPError, LlmResponse, TextResponse, http_error
+import json
+
+router = APIRouter()
+
+logger = logging.getLogger(__name__)
+
+RESPONSES = {
+    status.HTTP_200_OK: {"model": LlmResponse},
+    status.HTTP_400_BAD_REQUEST: {"model": HTTPError},
+    status.HTTP_401_UNAUTHORIZED: {"model": HTTPError},
+    status.HTTP_500_INTERNAL_SERVER_ERROR: {"model": HTTPError},
+}
+
+
+@router.post(
+    "/llm-generate",
+    response_model=LlmResponse,
+    responses=RESPONSES,
+    description="Generate text responses from input prompts using a large language model.",
+    operation_id="genLlm",
```

Comment:
@victorges can you check the SDK parameters are as expected 🙏🏻.

Reply:
Done! Sent inline comment above